Go instructions for ECS do not compile

File: [themes/default/content/docs/clouds/aws/guides/ecs.md](https://www.pulumi.com/docs/clouds/aws/guides/ecs/)

The go instructions do not seem to compile, I see the following:

```
% go build .
./main.go:8:2: ecs redeclared in this block
	./main.go:6:2: other declaration of ecs
./main.go:8:2: "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs" imported and not used
./main.go:9:2: lb redeclared in this block
	./main.go:7:2: other declaration of lb
./main.go:19:17: undefined: lb.NewApplicationLoadBalancer
./main.go:23:16: undefined: ecs.NewFargateService
./main.go:23:55: undefined: ecs.FargateServiceArgs
./main.go:27:29: undefined: ecs.FargateServiceTaskDefinitionArgs
./main.go:28:21: undefined: ecs.TaskDefinitionContainerDefinitionArgs
./main.go:33:26: undefined: ecs.TaskDefinitionPortMappingArgs
./main.go:34:12: undefined: ecs.TaskDefinitionPortMappingArgs
```
The import strings look suspicious to me and I believe are overlapping:

```
	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ecs"
	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/lb"
	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/lb"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
```

I'm new to Pulumi so I'm not entirely sure how to correct this issue but it does seem like aliasing the first import to `awsecs` and then using that to create the new cluster fixes several issues. Later on the `TaskDefinitionPortMappingArgs` don't have an obvious replacement.

--- a/themes/default/content/docs/clouds/aws/guides/ecs.md
+++ b/themes/default/content/docs/clouds/aws/guides/ecs.md
@@ -111,9 +111,8 @@ pulumi.export("url", lb.load_balancer.dns_name)
 package main
 
 import (
-	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ecs"
-	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/lb"
-	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
+	"github.com/pulumi/pulumi-aws/sdk/v6/go/aws/ecs"
+	xecs "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
 	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/lb"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
@@ -128,30 +127,32 @@ func main() {
 		if err != nil {
 			return err
 		}
-		_, err = ecs.NewFargateService(ctx, "service", &ecs.FargateServiceArgs{
+
+		tdpma := xecs.TaskDefinitionPortMappingArray{
+			xecs.TaskDefinitionPortMappingArgs{
+				TargetGroup: lb.DefaultTargetGroup,
+			},
+		}
+
+		_, err = xecs.NewFargateService(ctx, "service", &xecs.FargateServiceArgs{
 			Cluster:        cluster.Arn,
 			AssignPublicIp: pulumi.Bool(true),
 			DesiredCount:   pulumi.Int(2),
-			TaskDefinitionArgs: &ecs.FargateServiceTaskDefinitionArgs{
-				Container: &ecs.TaskDefinitionContainerDefinitionArgs{
-					Image:     pulumi.String("nginx:latest"),
-					Cpu:       pulumi.Int(512),
-					Memory:    pulumi.Int(128),
-					Essential: pulumi.Bool(true),
-					PortMappings: []ecs.TaskDefinitionPortMappingArgs{
-						&ecs.TaskDefinitionPortMappingArgs{
-							TargetGroup: lb.DefaultTargetGroup,
-						},
-					},
+			TaskDefinitionArgs: &xecs.FargateServiceTaskDefinitionArgs{
+				Container: &xecs.TaskDefinitionContainerDefinitionArgs{
+					Image:        pulumi.String("nginx:latest"),
+					Cpu:          pulumi.Int(512),
+					Memory:       pulumi.Int(128),
+					Essential:    pulumi.Bool(true),
+					PortMappings: tdpma,
 				},
 			},
 		})
+
 		if err != nil {
 			return err
 		}
-		ctx.Export("url", lb.LoadBalancer.ApplyT(func(loadBalancer *lb.LoadBalancer) (string, error) {
-			return loadBalancer.DnsName, nil
-		}).(pulumi.StringOutput))
+		ctx.Export("url", lb.LoadBalancer.DnsName())
 		return nil
 	})
 }
@@ -412,14 +413,14 @@ package main
 import (
 	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2"
 	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ecs"
-	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ec2"
-	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
+	xec2 "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ec2"
+	xecs "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
 func main() {
 	pulumi.Run(func(ctx *pulumi.Context) error {
-		vpc, err := ec2.NewVpc(ctx, "vpc", nil)
+		vpc, err := xec2.NewVpc(ctx, "vpc", nil)
 		if err != nil {
 			return err
 		}
@@ -446,7 +447,7 @@ func main() {
 		if err != nil {
 			return err
 		}
-		_, err = ecs.NewFargateService(ctx, "service", &ecs.FargateServiceArgs{
+		_, err = xecs.NewFargateService(ctx, "service", &xecs.FargateServiceArgs{
 			Cluster: cluster.Arn,
 			NetworkConfiguration: &ecs.ServiceNetworkConfigurationArgs{
 				Subnets: vpc.PrivateSubnetIds,
@@ -455,8 +456,8 @@ func main() {
 				},
 			},
 			DesiredCount: pulumi.Int(2),
-			TaskDefinitionArgs: &ecs.FargateServiceTaskDefinitionArgs{
-				Container: &ecs.TaskDefinitionContainerDefinitionArgs{
+			TaskDefinitionArgs: &xecs.FargateServiceTaskDefinitionArgs{
+				Container: &xecs.TaskDefinitionContainerDefinitionArgs{
 					Image:     pulumi.String("nginx:latest"),
 					Cpu:       pulumi.Int(512),
 					Memory:    pulumi.Int(128),
@@ -737,9 +738,8 @@ package main
 
 import (
 	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ecs"
-	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/lb"
 	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecr"
-	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
+	xecs "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
 	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/lb"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
@@ -750,44 +750,47 @@ func main() {
 		if err != nil {
 			return err
 		}
+
 		image, err := ecr.NewImage(ctx, "image", &ecr.ImageArgs{
 			RepositoryUrl: repository.Url,
-			Path:          pulumi.String("./app"),
 		})
 		if err != nil {
 			return err
 		}
+
 		cluster, err := ecs.NewCluster(ctx, "cluster", nil)
 		if err != nil {
 			return err
 		}
+
 		lb, err := lb.NewApplicationLoadBalancer(ctx, "lb", nil)
 		if err != nil {
 			return err
 		}
-		_, err = ecs.NewFargateService(ctx, "service", &ecs.FargateServiceArgs{
+
+		tdpma := xecs.TaskDefinitionPortMappingArray{
+			xecs.TaskDefinitionPortMappingArgs{
+				TargetGroup: lb.DefaultTargetGroup,
+			},
+		}
+		_, err = xecs.NewFargateService(ctx, "service", &xecs.FargateServiceArgs{
 			Cluster:        cluster.Arn,
 			AssignPublicIp: pulumi.Bool(true),
-			TaskDefinitionArgs: &ecs.FargateServiceTaskDefinitionArgs{
-				Container: &ecs.TaskDefinitionContainerDefinitionArgs{
-					Image:     image.ImageUri,
-					Cpu:       pulumi.Int(512),
-					Memory:    pulumi.Int(128),
-					Essential: pulumi.Bool(true),
-					PortMappings: []ecs.TaskDefinitionPortMappingArgs{
-						&ecs.TaskDefinitionPortMappingArgs{
-							TargetGroup: lb.DefaultTargetGroup,
-						},
-					},
+			TaskDefinitionArgs: &xecs.FargateServiceTaskDefinitionArgs{
+				Container: &xecs.TaskDefinitionContainerDefinitionArgs{
+					Image:        image.ImageUri,
+					Cpu:          pulumi.Int(512),
+					Memory:       pulumi.Int(128),
+					Essential:    pulumi.Bool(true),
+					PortMappings: tdpma,
 				},
 			},
 		})
 		if err != nil {
 			return err
 		}
-		ctx.Export("url", lb.LoadBalancer.ApplyT(func(loadBalancer *lb.LoadBalancer) (string, error) {
-			return loadBalancer.DnsName, nil
-		}).(pulumi.StringOutput))
+
+		ctx.Export("url", lb.LoadBalancer.DnsName())
 		return nil
 	})
 }

